tailcfg/tailcfg.go

```diff
@@ -1682,6 +1682,11 @@ const (
 	CapabilityIngress = "https://tailscale.com/cap/ingress"
 )
 
+const (
+	// NodeAttrFunnel grants the ability for a node to host ingress traffic.
+	NodeAttrFunnel = "funnel"
```

Comment:
Looks unused, remove? Maybe move comment to `CapabilityIngress` instead?

Reply:
oh, github fail. Can we reuse `CapabilityIngress` for the wire?

Reply:
in cli/serve.go
```go
if !slices.Contains(st.Self.Capabilities, tailcfg.NodeAttrFunnel) {
	return errors.New("Funnel not available. See https://tailscale.com/s/no-funnel")
}
```

Reply:
GitHub decided to hide most of the PR cause the diff was too big 🤦🏼‍♂️